bitcash/wallet.py

```diff
@@ -278,8 +282,11 @@ def send(self, outputs, fee=None, leftover=None, combine=True,
         :type combine: ``bool``
         :param message: A message to include in the transaction. This will be
                         stored in the blockchain forever. Due to size limits,
-                        each message will be stored in chunks of 40 bytes.
+                        each message will be stored in chunks of 220 bytes.
         :type message: ``str``
+        :param bytes: Whether or not the supplied message is already bytes. If
+                      false (default), message is automatically encoded to UTF-8.
+        :type bytes: ``bool``
```

Comment:
I think maybe `type` could make more sense here, or would automatic type detection be the Pythonic way?

I guess maybe it's not a good one to make automatic, but I think a typed input seems ideal where it just takes both types and processes them appropriately.

Reply:
I did a little Google searching at the time and gave up and just added this flag instead. But I'm looking at it more right now. The Pythonic way (duck typing) would be not to check that it is a specific type but, in this case, probably to see that it has the encode() function, as so:

    try:
        message = message.encode('utf-8')
    except AttributeError:
        pass # assume message is already a bytes-like object